bluebird.d.ts

```diff
@@ -151,8 +151,8 @@ declare class Bluebird<R> implements Bluebird.Thenable<R>, Bluebird.Inspection<R
    * });
    * </code>
    */
-  // TODO find way to fix get()
-  // get<U>(propertyName: string): Bluebird<U>;
+  // TODO: explain original comment from DT: TODO find way to fix get()
```

Comment:
I just checked the TS roadmap and couldn't find any mention of subtypes. Could you clarify what you mean ?